src/Illuminate/Testing/PendingCommand.php

```diff
@@ -86,6 +86,18 @@ public function expectsQuestion($question, $answer)
         return $this;
     }
 
+    /**
+     * Specify a confirmation question that should be asked when the command runs.
+     *
+     * @param  string  $question
+     * @param  string  $answer
```

Comment:
Why is this not allowing bool, but the above is?

Reply:
This method checks whether the answer is `yes`/`no`.

The method above also accepts a `bool` value as it is the way how to test confirmation questions.